Whispir_REST_API.postman_collection.json

```diff
@@ -7388,6 +7388,1340 @@
+							"host": [
+								"{{base-url}}"
+							],
+							"path": [
+								"otp",
+								"send-code"
+							]
+						}
+					},
+					"response": [
+						{
+							"name": "201: Success",
+							"originalRequest": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "x-api-key",
+										"value": "{{V5_API_KEY}}",
+										"type": "text"
+									},
+									{
+										"key": "whispirUserId",
+										"value": "{{USER_ID}}",
+										"type": "text",
+										"disabled": true
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"workspaceId\": \"{{TEST_WORKSPACE_ID}}\",\n    \"channel\": \"sms\",\n    \"phoneNumber\": \"+639075652923\",\n    \"codeLength\": 6,\n    \"expiresIn\": 18000\n}",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{BASE_URL}}/send-code",
+									"host": [
+										"{{BASE_URL}}"
+									],
+									"path": [
+										"send-code"
+									]
+								}
+							},
+							"status": "Created",
+							"code": 201,
+							"_postman_previewlanguage": "json",
+							"header": [
+								{
+									"key": "Content-Type",
+									"value": "application/json"
+								},
+								{
+									"key": "Content-Length",
+									"value": "238"
+								},
+								{
+									"key": "Connection",
+									"value": "keep-alive"
+								},
+								{
+									"key": "Date",
+									"value": "Wed, 05 Jul 2023 14:02:28 GMT"
+								},
+								{
+									"key": "x-amzn-RequestId",
+									"value": "6ac5ff6a-fc8f-445e-9001-685de8cb883c"
+								},
+								{
+									"key": "x-amzn-Remapped-x-amzn-RequestId",
+									"value": "1117e97b-0ffd-4006-85f8-2c4edb73e4cc"
+								},
+								{
+									"key": "x-amzn-Remapped-Content-Length",
+									"value": "238"
+								},
+								{
+									"key": "x-amzn-Remapped-Connection",
+									"value": "keep-alive"
+								},
+								{
+									"key": "Via",
+									"value": "1.1 0e61cdf08a154ac7d647c2dc742467a6.cloudfront.net (CloudFront), 1.1 933f4fb3c49c9a37f5b5461deacd212c.cloudfront.net (CloudFront)"
+								},
+								{
+									"key": "x-amz-apigw-id",
+									"value": "Hl-uNG3bywMFiZQ="
+								},
+								{
+									"key": "X-Amzn-Trace-Id",
+									"value": "Root=1-64a577f4-5fa1a8574f8506495769430d;Sampled=0;lineage=a57e7ed2:0"
+								},
+								{
+									"key": "X-Amz-Cf-Pop",
+									"value": "SYD62-P2"
+								},
+								{
+									"key": "X-Amz-Cf-Pop",
+									"value": "HKG60-C1"
+								},
+								{
+									"key": "x-amzn-Remapped-Date",
+									"value": "Wed, 05 Jul 2023 14:02:28 GMT"
+								},
+								{
+									"key": "X-Cache",
+									"value": "Miss from cloudfront"
+								},
+								{
+									"key": "X-Amz-Cf-Id",
+									"value": "diqYjsEeyOT1GK1lzT9lszKl4YDrL1gI020nCl1lPa28hXlgHyGqWA=="
+								}
```

Comment:
Are we not supposed to mask this @Fatal-Errol ?

Reply:
Hi @gabzi-whispir, I was instructed to remove this to make it consistent with the other endpoints.

Reply:
That's right. Apologies if I didn't post my request here for visiblity. Thanks team 🙏